<ui version="4.0" >
 <class>NewVersionAvailableDialogClass</class>
 <widget class="QDialog" name="NewVersionAvailableDialogClass" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WindowTitle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="sizeConstraint" >
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="frameVLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="descriptionBox" >
          <property name="tabChangesFocus" >
           <bool>true</bool>
          </property>
          <property name="openExternalLinks" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="downloadButton" >
            <property name="accessibleName" >
             <string>rectButton</string>
            </property>
            <property name="text" >
             <string>DownloadButton</string>
            </property>
            <property name="default" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeButton" >
            <property name="accessibleName" >
             <string>rectButton</string>
            </property>
            <property name="text" >
             <string>CloseButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
